### ISSUE TITLE
time cursor overlaps sidebar when should be offscreen / hidden

### ISSUE THREAD

**Issue by colehart** [samuelarbibe/dnd-timeline#36]
Hello!

I'm working on an implementation that uses the time cursor, and I noticed that there's a bug where the time cursor still shows up when the timeframe is scrolled to the right past where it should show up anymore and it overlaps the sidebar. It exists in the example as well

![Screenshot 2024-06-25 at 2 52 23 PM](https://github.com/samuelarbibe/dnd-timeline/assets/36777482/da8c9516-4ddb-41e9-b726-ddc2928caf18)

Any suggestions for a fix?

**Comment by colehart** [comment]
Actually figured it out. you just have to manipulate the visibility based on the timeDelta's negative value. Here's what I did to get it working:

on [this line](https://github.com/samuelarbibe/dnd-timeline/blob/814a193648fb051a792273739ca6206025de81fe/examples/timeaxis/src/TimeCursor.tsx#L21), add the following: 
```javascript
      if (timeDelta < 0) {
        timeCursorRef.current.style.visibility = 'hidden';

        return;
      };
 ```
 
 then below that you have to make it visible again for when you scroll back to a timeframe where now is in view. you could add this to the bottom of the offset cursor function to make it visible again:
 ```javascript
      timeCursorRef.current.style.visibility = 'visible';
 ```

**Comment by samuelarbibe** [comment]
Thanks for the heads up

Fixed in the examples here: #37
